src/modules/user/controllers/controllerRegister.js

```diff
@@ -54,17 +53,13 @@ export const handleRegister = async function (req, res) {
       }
     }
 
-    const result = await databasePrisma.user.create({
+    await databasePrisma.user.create({
       data,
     });
```

Comment:
add the user to the request
```suggestion
    });
    req.user = user
```